client/utils.ts

```diff
@@ -25,11 +25,14 @@ export function redirectToLaunchpad(
 	launchpadFlow: string,
 	emailVerifiedParam: boolean
 ) {
-	let launchpadRedirectionURL = addQueryArgs('/setup/launchpad', {
-		flow: launchpadFlow,
-		siteSlug,
-		emailVerified: emailVerifiedParam ? 1 : undefined
-	});
+	const launchpadRedirectionURL = addQueryArgs(
+		{
+			flow: launchpadFlow,
+			siteSlug,
+			emailVerified: emailVerifiedParam ? 1 : undefined,
```

Comment:
just curious why do we have to explicitly return undefined

Reply:
`addQueryArgs` will remove any undefined query args from the new URL it creates.